src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -45,7 +45,8 @@ interface IBlazor {
     readSatelliteAssemblies?: () => System_Array<System_Object>,
     getLazyAssemblies?: any
     dotNetCriticalError?: any
-    getSatelliteAssemblies?: any
+    getSatelliteAssemblies?: any,
+    applyHotReload?: (id: string, metadataDelta: string, ilDeta: string) => void
```

Comment:
```suggestion
    applyHotReload?: (id: string, metadataDelta: string, ilDelta: string) => void
```

Reply:
Also should `ilDelta` maybe be a `Uint8Array` or something like that instead of `string`? That would give us the flexibility to put in more efficient ways to transport the data into the JS runtime in the future.

Reply:
Actually I don't get why `metadataDelta` and `ilDelta` are typed as `string` here at all. They are declared as `byte[]` in the .NET code. Looks like we're relying on the absence of runtime type checks. Can we make the declarations consistent?

Reply:
We're currently serializing byte[] as base64 encoded strings. S.T.J does the transformation for us. I'll file a follow up so we can make sure we're using the base format to send these deltas across. That said, at least in my very minimal demos, these didn't have an appearance of being slow.